se/spelling.py

```diff
@@ -341,7 +341,7 @@ def modernize_spelling(xhtml: str) -> str:
 	xhtml = regex.sub(r"\b([Aa])ny where\b", r"\1nywhere", xhtml)			# any where -> anywhere
 	xhtml = regex.sub(r"\b([Ee])very thing\b", r"\1verything", xhtml)		# every thing -> everything
 	xhtml = regex.sub(r"\b([Aa])ny thing\b", r"\1nything", xhtml)			# any thing -> anything
-	xhtml = regex.sub(r"\b([Rr])e-enforcement", r"\1inforcement", xhtml)		# re-enforcement -> reinforcement
+	xhtml = regex.sub(r"\b([Rr])e-?enforce(ment)?", r"\1inforce\2", xhtml)		# re-enforce[ment] -> reinforce[ment]
```

Comment:
I don't think you need to include the 2nd capture group. Just leave it as `-?enforce` because we don't really care about the suffix.